lib/formtools/overview.js

```diff
@@ -0,0 +1,183 @@
+
+var async = require('async'),
+    cellRenderers = require('./renderers-cell');
+
+/**
+ * Set unique tabId to each tab in overview.body
+ * @param {Array} overviewBody overview body
+ */
+function setTabId (overviewBody) {
+
+    overviewBody.forEach(function (section, sectionIndex) {
+
+        if (Array.isArray(section)) {
+
+            section.forEach(function (tab, tabIndex) {
+
+                // Replace white space from tab.lable with "-", make it lowecase and add the index number at the end.
+                // sectionIndex and tabIndex number are added at the end to make the tabID unique in case when more then one tab have the same label (title)
+                tab.tabId = (tab.label).replace(/\s+/g, '-').toLowerCase() + '-' + sectionIndex + '-' + tabIndex;
+
+            });
+        }
+    });
+
+}
+
+/*
+ * Replace each field name in section of overview.body with object containing field label and filed value
+ * e.g ({ label: '', value: '' })
+ * @param  {Object}   record   Overview record object
+ * @param  {Object}   model    Model object
+ * @param  {Object}   section  Section Object with its label and an array of fileds (e.g { label: '', fields: [] })
+ * @param  {Function} callback Callback function
+ * @return {[object]}          Return object: { label: '', fields: [ { label: '', value: '' }, ... ] } to callback
+ */
+function renderFields(record, model, section, callback) {
+
+    var fieldset = {
+        label: section.label,
+        fields: []
+    };
+
+    async.eachSeries(section.fields, function (field, cb) {
```

Comment:
Perhaps use `async.mapSeries` here which returns an array you don't need to manually push it into `fields`